### DISCUSSION TITLE
Wondering for built-in "Tree style tab" or Chromium "group tabs"

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#878] [Ideas]
### What feature would you like to see?

I'm just tried the zen browser and loved that built-in vertical tab, but if there are something like  built-in Tree style tab or group tabs, it would be perfect:)

### What platform are you seeing the problem on?

Windows

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/231_

**Comment by GitMurf** [comment]
FYI see here for a Discussion one of the maintainers opened up to follow this development: https://github.com/zen-browser/desktop/discussions/891

**Comment by guigirl42** [comment]
https://github.com/zen-browser/desktop/discussions/891#discussioncomment-11288211
